include/deal.II/lac/parallel_vector.h

```diff
@@ -1257,7 +1257,7 @@ namespace parallel
     Vector<Number> &
     Vector<Number>::operator = (const Vector<Number> &c)
     {
-      return this->template operator=<Number>(c);
+      return this->operator=<Number>(c);
```

Comment:
I'd guess that the majority of compilers on *nix won't be very happy with the template being removed.